.travis.yml

```diff
@@ -89,35 +90,48 @@ install:
   ;;
   esac
 - cargo install --list
+
 script: |
   case "$FEATURE" in
 
   "fmt" )
       cd exonum && cargo update && cd .. &&
       cd sandbox && cargo update && cd .. &&
+      cd testkit && cargo update && cd .. &&
```

Comment:
There's a single `lockfile` per workspace, so just `cargo update && cd sandbox && cargo update` should suffice.